src/syft/util.py

```diff
@@ -58,7 +58,7 @@ def get_subclasses(obj_type: type) -> List[type]:
 
     """
 
-    classes = list()
+    classes = []
```

Comment:
They have nothing to do with the PR description.
Those were some simple fixes shown by flake